### ISSUE TITLE
add chinese doc

### ISSUE THREAD

**Issue by chiangbar** [sparckles/Robyn#999]
Can you add Chinese documentation so that more developers who understand Chinese can use and promote this web framework? It is not accessible on the Robyn official website in China

**Comment by VishnuSanal** [comment]
we currently do not support i18n, AFAIK. this would require significant changes in the codebase too. nextjs has native support for ii18n (https://nextjs.org/docs/pages/building-your-application/routing/internationalization) & should figure out whether we can outsource the translations using something like weblate (ref: https://docs.weblate.org/en/latest/formats/i18next.html). inviting @sansyrox's opinions.

**Comment by sansyrox** [comment]
Hey @chiangbar @VishnuSanal  👋 

Unfortunately, we don't have i18n on Robyn right now. But I would love to add it as a part of the docs. Let me go through the docs and get back to you both 😄

**Comment by chiangbar** [comment]
> we currently do not support i18n, AFAIK. this would require significant changes in the codebase too. nextjs has native support for ii18n (https://nextjs.org/docs/pages/building-your-application/routing/internationalization) & should figure out whether we can outsource the translations using something like weblate (ref: https://docs.weblate.org/en/latest/formats/i18next.html). inviting @sansyrox's opinions.

This has no impact on the code repository. What I want to say is to add a Robyn Chinese usage document

**Comment by sansyrox** [comment]
@chiangbar , 
> This has no impact on the code repository. What I want to say is to add a Robyn Chinese usage document

I agree and I'd love to add it. We will just have to change a few lines of code in the website to show chinese and english alongside each other.
